tckprocess.adoc

```diff
@@ -0,0 +1,422 @@
+contributors you have met the intended TCK requirements and is just one of
+the requirements for logo usage. Logos only apply to the platform specification.
+
+[#_filing_a_certification_request]
+### Filing a Certification Request
+
+Requests to be acknowledged as a certified implementation MUST be filed
+via the specification's issue tracker using the label
+`certification` and include the following information:
+
+-   Statement of Acceptance of the terms of the EFTL
+
+-   Product Name, Version and download URL (if applicable)
+
+-   Specification Name, Version and download URL
+
+-   TCK Version, digital SHA-256 fingerprint and download URL
+
+-   Implementation runtime Version(s) tested
+
+-   Public URL of TCK Results Summary
+
+-   Any Additional Specification Certification Requirements
+
+-   Java runtime used to run the implementation
+
+-   Summary of the information for the certification environment,
+    operating system, cloud, ...​
+
+-   A statement attesting that all TCK requirements have been met,
+    including any compatibility rules
+
+[#_additional_specification_certification_requirements]
+### Additional Specification Certification Requirements
+
+Specification's may require additional items for a Certification
+Request as defined in their corresponding TCK Documentation under the
+section labeled \"Additional Certification Requirements\".
+
+Examples of such additional requirements may include:
+
+-   Name and version of Compatible Implementation used for
+    interoperability tests
+
+-   Name and version of Databases used for persistence tests
+
+-   Name and version of NoSQL implementations used in persistence tests
+
+[#_public_tck_results_summary]
+### Public TCK Results Summary
+
+While certification is on your honor, the community MUST be able to see
+your test results summary. At a minimum a results summary MUST:
+
+-   Be publicly visible with no password protection or sign-up
+
+-   Include a Summary Page containing:
+
+    -   All information in the above Certification Request
+
+    -   The Total number of tests run and passed. If you are running a
+        platform TCK, this means reporting the tests run for each
+        required TCK, which includes the platform TCK and some of the
+        individual TCKs that are not integrated with the platform TCK.
+
+An optional "Test List Page" showing all tests run may be linked from
+the Summary Page. The Summary Page URL is the URL that MUST be included
+in any Certification Requests.
+
+The following are explicitly not requirements:
+
+-   The Ability for the public to run the tests themselves
+
+-   Full log output of the TCK
+
+Implementors may supply this information and provide support for how to
+run a TCK against their implementation, but it is not required.
+
+[#_certification_resolution]
+### Certification Resolution
+
+As noted, certification requests do not in themselves grant rights to
```

Comment:
Does this mean: create a CCR and then start a ballot on microprofile-wg mailinglist? Can we spell this out clearly?

Reply:
Don't need a ballot voting.

Reply:
If -1 is filed, simply majority policy will be used.

Reply:
I will update this section not to mention logo but having the compatible impl listed on the compatibility page we are going to create